packages/celotool/src/lib/geth.ts

```diff
@@ -628,10 +638,18 @@ export const simulateClient = async (
         ...baseLogMessage,
       })
     }
-
+    const dataStr = dataTest ? getBigData(119750) : undefined // aim for half million gas txs
```

Comment:
Don't know if `119750` depends on something or not, or it just making the calculation on the 4 gas per bytes, but I would extract this as a constant that explains a little more the number
It seems that the calculation is (500k - 21k)/4
So I would add this:
```
const BASE_TX_GAS = 21000
const GAS_PER_BYTE = 4
const BYTES_FOR_500k_GAS_BASE_TX = (500000 - BASE_TX_GAS) / GAS_PER_BYTE
```
This way, it's not only clear why that number, but also, if we change any of those params in a fork, we can easily fix this test

Reply:
yes, it's that calculation exactly. I'll clarify that.